.github/workflows/simulation-1.2.yml

```diff
@@ -93,3 +93,49 @@ jobs:
     - name: Codecov
       run: |
         bash <(curl -s https://codecov.io/bash) -x "${{ matrix.compiler.gcov }}" -Z
+
+  thread-1-2-backbone:
+    runs-on: ubuntu-18.04
+    env:
+      REFERENCE_DEVICE: 1
+      VIRTUAL_TIME: 0
+      PACKET_VERIFICATION: 1
+      THREAD_VERSION: 1.2
+      INTER_OP: 1
+      COVERAGE: 1
+      MULTIPLY: 3
+      PYTHONUNBUFFERED: 1
+      OTBR_COMMIT: "ea9cd69aa2eed3f320f6a80a4d0b966959c19ff0" # Wed Sep 2 09:10:00 2020 +0800
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build OTBR Docker
+        run: |
+          ./script/test build_otbr_docker
+      - name: Bootstrap
+        run: |
+          sudo rm /etc/apt/sources.list.d/* && sudo apt-get update
+          sudo apt-get --no-install-recommends install -y python3-setuptools python3-wheel ninja-build socat curl ca-certificates
+          python3 -m pip install -r tests/scripts/thread-cert/requirements.txt
+      - name: Build
+        run: |
+          ./script/test build
+      - name: Get Thread-Wireshark
+        run: |
+          ./script/test get_thread_wireshark
+      - name: Run
+        run: |
+          export CI_ENV="$(bash <(curl -s https://codecov.io/env)) -e GITHUB_ACTIONS"
```

Comment:
Explain: `CI_ENV` gets all environment variables that need to be passed to OTBR Docker for Codecov Uploading.